ent/schema/planet.go

```diff
@@ -90,6 +90,9 @@ func (Planet) Fields() []ent.Field {
 	return []ent.Field{
 		field.String("name").
 			Default("Unnamed"),
+		field.Enum("planet_type").
```

Comment:
You can also add `.Immutable()` to this as it should not be changed after the planet is created. Same for the position code, orbits, etc.